src/lmn-gtm-analytics.js

```diff
@@ -78,6 +78,9 @@ const lmnAnalytics = {
       if (callback) {
```

Comment:
Indentation seems to have gone off here. Mind fixing it while you're in the code?